File: Programmers/Inho/C#/Level01/제일 작은 수 제거하기.cs
using System;
using System.Linq;
using System.Collections.Generic;
public class Solution {
    public int[] solution(int[] arr) {
        int[] answer = new int[] {};
        if(arr.Length == 1){
            answer = new int[] {-1};
            return answer;
        }
        int small = arr[0];
        foreach(int i in arr){
            if(small>i)
                small = i;
        }
        answer = arr.Where(condition => condition!=small).ToArray();
        return answer;
    }
}

File: Programmers/Inho/C++/Level01/제일 작은 수 제거하기.cpp
#include <string>
#include <vector>
#include <iostream>
using namespace std;

vector<int> solution(vector<int> arr) {
    vector<int> answer;
    int min = arr[0];
    if(arr.size()==1){
        answer.push_back(-1);
        return answer;
    }
        
    for(int i = 0;i<arr.size();i++){
        if(arr[i]<min)
            min = arr[i];
    }
    
    for(int i = 0;i<arr.size();i++){
        if(arr[i]==min)
            continue;
        answer.push_back(arr[i]);
    }
    return answer;
}